File: src/main/java/net/dontdrinkandroot/gitki/service/markdown/GitkiAttributeProviderFactory.java
package net.dontdrinkandroot.gitki.service.markdown;

import org.commonmark.renderer.html.AttributeProvider;
import org.commonmark.renderer.html.AttributeProviderContext;
import org.commonmark.renderer.html.AttributeProviderFactory;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class GitkiAttributeProviderFactory implements AttributeProviderFactory
{
    @Override
    public AttributeProvider create(AttributeProviderContext context)
    {
        return new TableAttributeProvider();
    }
}

File: src/main/kotlin/net/dontdrinkandroot/gitki/service/markdown/GitkiAttributeProviderFactory.kt
package net.dontdrinkandroot.gitki.service.markdown

import org.commonmark.renderer.html.AttributeProvider
import org.commonmark.renderer.html.AttributeProviderContext
import org.commonmark.renderer.html.AttributeProviderFactory

class GitkiAttributeProviderFactory : AttributeProviderFactory {

    override fun create(context: AttributeProviderContext): AttributeProvider {
        return TableAttributeProvider()
    }
}